yt_dlp/extractor/fujitv.py

```diff
@@ -31,6 +38,8 @@ def _real_extract(self, url):
         video_id = self._match_id(url)
         formats = self._extract_m3u8_formats(
             self._BASE_URL + 'abr/tv_android/%s.m3u8' % video_id, video_id, 'mp4')
+        json_info = self._download_json('https://fod-sp.fujitv.co.jp/apps/api/episode/detail/?ep_id=%s&is_premium=false' % video_id, video_id, headers={'x-authorization': 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJGT0QiLCJ1aWQiOiIiLCJkdl90eXBlIjoid2ViIiwiZHZfaWQiOiJNaWNyb3NvZnQgV2luZG93cyBOVCAxMC4wLjE0MzkzLjBfQ2hyb21lOTYuMCJ9.tmK0o79qfDaBNVm7prjCG39DYvldacx8t38WA7TRlBA'})
```

Comment:
Are you sure the bearer token is safe to hardcode

Reply:
```suggestion
        self._download_webpage(url, fatal=False)
        token = self._get_cookies(url).get('CT').value or 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJGT0QiLCJ1aWQiOiIiLCJkdl90eXBlIjoid2ViIiwiZHZfaWQiOiJNaWNyb3NvZnQgV2luZG93cyBOVCAxMC4wLjE0MzkzLjBfQ2hyb21lOTYuMCJ9.tmK0o79qfDaBNVm7prjCG39DYvldacx8t38WA7TRlBA'
        json_info = self._download_json('https://fod-sp.fujitv.co.jp/apps/api/episode/detail/?ep_id=%s&is_premium=false' % video_id, video_id, headers={'x-authorization': 'Bearer ' + token})
```

Reply:
definitely not tobe hardcoded, just don't know where the token came from

Reply:
`self._get_cookies(url)`
this returned None

Reply:
If the token was hardcoded in the site, only then hardcode it.
Otherwise just rely on the cookies.

Reply:
this is it's payload
```
{
  "iss": "FOD",
  "uid": "",
  "dv_type": "web",
  "dv_id": "Microsoft Windows NT 10.0.14393.0_Chrome96.0"
}
```
It can break for linux or macos users (untested)

Reply:
because the `dv_id`, that may break, that's why stored in cookie, just use the token in cookie